app/views/location/list_locations.html.erb

```diff
@@ -1,68 +1,71 @@
 <%
-  new_tab_set(:sort_by_header.t, @sorts)
-  new_tab_set do
-    add_tab_with_query(:list_place_names_map.t, :action => :map_locations)
-    add_tab(:list_countries.t, :action => :list_countries)
-    add_tab_with_query(:show_location_create.t, :action => 'create_location')
-    add_tabs(@links)
-  end
+   new_tab_set(:sort_by_header.t, @sorts)
+   new_tab_set do
+     add_tab_with_query(:list_place_names_map.t, :action => :map_locations)
+     add_tab(:list_countries.t, :action => :list_countries)
+     add_tab_with_query(:show_location_create.t, :action => 'create_location')
+     add_tabs(@links)
+   end
 
-  @known_pages = @pages
-  @known_data = @objects
+   @known_pages = @pages
+   @known_data = @objects
 
-  if (!@known_pages or @known_pages.empty?) and
-     (!@undef_pages or @undef_pages.empty?)
-    flash_error(@error) if @error
-  end
+   if (!@known_pages or @known_pages.empty?) and
```

Comment:
Is it possible to use `try` here &/or other places?  See [try(*args)](http://api.rubyonrails.org/classes/NilClass.html#method-i-try).
(I haven't played around with the try method yet.)